tests/gamma/csm_observability_test.py

```diff
@@ -45,39 +92,383 @@ def setUpClass(cls):
         super().setUpClass()
         cls.metric_client = cls.gcp_api_manager.monitoring_metric_service("v3")
 
+    # A helper function to make the cloud monitoring API call to query metrics
+    # created by this test run.
+    def _query_metrics(self, metric_names, filter_str, interval):
+        results = {}
+        for metric in metric_names:
+            response = self.metric_client.list_time_series(
+                name=f"projects/{self.project}",
+                filter=filter_str
+                % {
+                    "metric": metric,
+                    "grpc_method": _GRPC_METHOD_NAME,
+                },
+                interval=interval,
+                view=monitoring_v3.ListTimeSeriesRequest.TimeSeriesView.FULL,
+            )
+            time_series = []
+            logger.debug("Metric: %s", metric)
+            for series in response:
+                logger.debug(" Metric Labels:")
+                for label_key, label_value in series.metric.labels.items():
+                    logger.debug(
+                        "  %(label_key)s: %(label_value)s",
+                        {
+                            "label_key": label_key,
+                            "label_value": label_value,
+                        },
+                    )
+                logger.debug(" Resource Labels:")
+                for label_key, label_value in series.resource.labels.items():
+                    logger.debug(
+                        "  %(label_key)s: %(label_value)s",
+                        {
+                            "label_key": label_key,
+                            "label_value": label_value,
+                        },
+                    )
+                time_series.append(series)
+            results[metric] = time_series
+        return results
+
+    # A helper function to check whether at least one of the "points" whose
+    # mean should be within 5% of ref_bytes.
+    def _at_least_one_point_within_range(self, points, ref_bytes) -> bool:
```

Comment:
Call it `assertAtLeastOnePointWithinRange` and assert directly in the method.
Then instead of 

```py
self.assertTrue(self._at_least_one_point_within_range(...))
```

you'll be able to do 

```py
self.assertAtLeastOnePointWithinRange(...)
```

Reply:
Done.